switcheroo/server/retrieve_public_keys.py

```diff
@@ -21,7 +21,7 @@ def get_public_keys_s3(connecting_user: str, bucket_name: str) -> str:
     host_name = socket.getfqdn()
     s3_client = boto3.client("s3")
```

Comment:
this should be done via the datastore (using AWS should be done only when the AWS S3 data store is in use)